### PR TITLE
Customize cosmos config for mesg engine

### LINKED ISSUE #1571: Change cosmos prefix address to mesgtest

### PR DESCRIPTION
**Pull request by krhubert**
close #1571 

Modified address for cosmos config (see: https://github.com/cosmos/cosmos-sdk/blob/master//types/address.go#L18-L69)

Also, we should reserve a code for mesg tokens according to https://github.com/satoshilabs/slips/blob/master/slip-0044.md

### CODE REVIEW

**Comment by krhubert** [comment]
2nd time the e2e test fails, that's something suspicious about that ... not sure why

**Comment by antho1404** [comment]
Please have a look at this PR https://github.com/mesg-foundation/engine/pull/1575. It should fix the issue.
I don't know why we didn't see that earlier tho...

File: cosmos/config.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	Bech32PrefixValAddr = Bech32MainPrefix + PrefixValidator + PrefixOperator
+	// Bech32PrefixValPub defines the Bech32 prefix of a validator's operator public key
+	Bech32PrefixValPub = Bech32MainPrefix + PrefixValidator + PrefixOperator + PrefixPublic
+	// Bech32PrefixConsAddr defines the Bech32 prefix of a consensus node address
+	Bech32PrefixConsAddr = Bech32MainPrefix + PrefixValidator + PrefixConsensus
+	// Bech32PrefixConsPub defines the Bech32 prefix of a consensus node public key
+	Bech32PrefixConsPub = Bech32MainPrefix + PrefixValidator + PrefixConsensus + PrefixPublic
+)
+
+func CustomizeConfig() {
```
Comment (antho1404) [inline cosmos/config.go]:
```suggestion
// CustomizeConfig customizes the cosmos application like addresses prefixes and coin type
func CustomizeConfig() {
```

File: cosmos/config.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package cosmos
+
+import sdktypes "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	// CoinType is a mesg coin type.
+	CoinType = 50000
+
+	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address
```
Comment (antho1404) [inline cosmos/config.go]:
```suggestion
	// Bech32MainPrefix defines the main Bech32 prefix
```

**Pull request merged**